app/models/recommendation.rb

```diff
@@ -24,4 +24,12 @@ class Recommendation < VersionedRecord
 
   validates :title, presence: true
   validates :reference, presence: true, uniqueness: true
+
+  SUPPORT_LEVELS = {
+    noted: 0,
+    supported_in_part: 1,
+    supported: 2
+  }
```

Comment:
nice! at first I was a bit sceptical about the need to specify the values here, but happy to keep this

Reply:
@lukearndt thought these were only internal and not exposed via the API which is definitely not what we want - instead i want the API to work with the integer values

cc @parndt

Reply:
so yeah please drop these named levels and use simple integer instead - no need for using an enum

Reply:
nevermind - managed to removed these on my own